pkg/mimirpb/query_response_extra_test.go

```diff
@@ -86,73 +86,5 @@ func TestFloatHistogramProtobufTypeRemainsInSyncWithPrometheus(t *testing.T) {
 	protoType := reflect.TypeOf(FloatHistogram{})
 	prometheusType := reflect.TypeOf(histogram.FloatHistogram{})
 
-	requireSameShape(t, prometheusType, protoType)
```

Comment:
Might be worth moving a version of the "why does this test exist" comment above to the `RequireSameShape` method in `util/test/shape.go`.